index.html

```diff
@@ -25,7 +25,7 @@
 <script type="text/template" id="dbox-donation-template">
   <section class="dbox-donation-section">
     <h2>Donate to devanooga</h2>
-    <p>devanooga is looking to work on many fun projects and events, become a donor today!</p>
+    <p>devanooga is looking to work on many fun projects and events, become a donor today! devanooga is a 501(c)(3) and your donations are tax deductible.</p>
```

Comment:
On second thought, let’s change the first sentence to start with “we’re” (to avoid repetition):
>We’re looking to work on many fun projects and events; become a donor today! devanooga is a 501(c)(3), and your donation is tax deductible.